frontend/src/components/Forms/ShoutoutsPage/ShoutoutCard.tsx

```diff
@@ -1,30 +1,86 @@
-import { Card } from 'semantic-ui-react';
+import React, { useState } from 'react';
+import { Button, Card, Form, Icon, Modal, TextArea } from 'semantic-ui-react';
+import ShoutoutsAPI from '../../../API/ShoutoutsAPI';
 import ShoutoutDeleteModal from '../../Modals/ShoutoutDeleteModal';
+import { Emitters } from '../../../utils';
 import styles from './ShoutoutCard.module.css';
 
-const ShoutoutCard = (props: {
-  shoutout: Shoutout;
-  setGivenShoutouts: React.Dispatch<React.SetStateAction<Shoutout[]>>;
-}): JSX.Element => {
-  const { shoutout, setGivenShoutouts } = props;
+const ShoutoutCard = ({ shoutout, setGivenShoutouts }) => {
```

Comment:
Make sure to give a type to `shoutout` and `setGivenShoutouts` (like the original code) to pass the type check.